buildtest/settings/config.yml

```diff
@@ -1,45 +1,23 @@
 system:
   generic:
-    # specify list of hostnames where buildtest can run for given system record
-    hostnames: [".*"]
-
-    # system description
```

Comment:
@kingxleyy this was probably my mistake. Please revert this change back to original file shown in `devel` branch. Just copy the entire file and commit it locally and push it so there are no changes shown for this file

Reply:
done